boltzrpc/boltzrpc.proto

```diff
@@ -733,6 +738,39 @@ message GetWalletRequest {
     optional uint64 id = 2;
 }
 
+message ListWalletTransactionsRequest {
+    uint64 id = 1;
+    optional bool exclude_swap_related = 2;
+}
+
+enum TransactionType {
+    ANY = 0;
```

Comment:
What does `ANY` mean?

Reply:
Thought it could ne useful if the tx type cant be identified clearly